#642 | Command GetKeysAndFlags Implementation

YuvrajGosain opened the pull request:
This Closes Issue - #642

apoorvyadav1111 commented:
Hi @YuvrajGosain, Thanks for the changes, Could you please rebase and check why the linter is failing?

YuvrajGosain commented:
> Hi @YuvrajGosain, Thanks for the changes, Could you please rebase and check why the linter is failing?

Hi @apoorvyadav1111 can you pls help review this. I have rebased and addressed linting issues.

JyotinderSingh commented:
Looks like this PR is out of date, could you please rebase this on the latest master?

arpitbbhayani commented:
@YuvrajGosain can you please rebase?

YuvrajGosain commented:
Hi @arpitbbhayani and @JyotinderSingh sure would rebase this.

apoorvyadav1111 commented:
Hi @YuvrajGosain , How are you? I am excited to know if you have any updates on this issue.

arpitbbhayani commented:
this is obsolete now. Closing the PR to keep things clean. Thanks for working on the patch.
